src/Keboola/StorageApi/Components.php

```diff
@@ -329,18 +329,47 @@ public function createConfigurationRowFromVersion($componentId, $configurationId
         );
     }
 
+    /**
+     * @param int $id
+     * @return array
+     */
+    public function resetWorkspacePassword($id)
+    {
+        $result = $this->client->apiPostJson("workspaces/{$id}/password");
+        assert(is_array($result));
+        return $result;
+    }
+
     public function createConfigurationWorkspace($componentId, $configurationId, array $options = [], bool $async = false)
     {
         $url = $this->branchPrefix . "components/{$componentId}/configs/{$configurationId}/workspaces";
         if ($async) {
             $url .= '?' . http_build_query(['async' => true]);
         }
-        return $this->client->apiPostJson(
+        $result = $this->client->apiPostJson(
             $url,
             $options,
             true,
             [Client::REQUEST_OPTION_EXTENDED_TIMEOUT => true]
         );
+        assert(is_array($result));
+
+        $response = $this->resetWorkspacePassword($result['id']);
```

Comment:
Nemůžeš si udělat new Workspaces($this->client) a volat to na tom?

Reply:
mas pravdu, opraveno